src/EventListener/LogRevisionsListener.php

```diff
@@ -161,16 +161,8 @@ public function postFlush(PostFlushEventArgs $eventArgs): void
                     if (isset($meta->fieldMappings[$idField])) {
                         $columnName = $meta->fieldMappings[$idField]['columnName'];
                         $types[] = $meta->fieldMappings[$idField]['type'];
-                    } elseif (isset($meta->associationMappings[$idField])) {
-                        $columnName = $meta->associationMappings[$idField]['joinColumns'][0];
-                        if (\is_array($columnName)) {
```

Comment:
A bugfix was introduce 6 years ago because it's an array, but in fact it's always an array.

Reply:
Can you link the PR? is the discussion there relevant?

Reply:
Not really relevant imho.
Someone got an error and add an in array check which was always true.

I can find the PR but I think it was quickly merged without discussion